### PR TITLE
Add planning high and penalty high limits for ACA xija model

### PR DESCRIPTION
**Pull request by taldcroft**
This implements the change in the ACA planning limit that was approved at SS&AWG 6/23 and at MPCWG 6/16.

This is a precursor for corresponding updates in proseco and potentially MATLAB tools to use the new `limits` values.

## Testing

- [x] Functional testing with xija gui fit to see the limit lines drawn in the expected place.

image

### CODE REVIEW

**Review by jzuhone — APPROVED** [review]

**Review by matthewdahmer — APPROVED** [review]
I added code that can help determine if two versions of a given model spec result in identical predictions. Running this code indicates the proposed ACA model produces identical results to the version included in Release 3.35.

File: chandra_models/compare_models.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
+from hashlib import md5
+import json
+from urllib.request import urlopen
+import xija
+
+
+def get_github_model_spec(branch, directory, filename):
```
Comment (taldcroft) [inline chandra_models/compare_models.py:7]: You should be using [get_xija_model_spec](https://sot.github.io/xija/api.html#xija.get_model_spec.get_xija_model_spec) which does the same thing (where the `repo_path` can be the GitHub URL).

File: chandra_models/compare_models.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    current_spec, current_md5 = get_github_model_spec(branch_1, directory, filename)
+    print(f'Running baseline model for {directory_name}/{file_name}, MD5 hash:{current_md5}')
+    current_predictions = run_xija_model(current_spec, tstart, tstop, msid, init)
+
+    new_spec, new_md5 = get_github_model_spec(branch_2, directory, filename)
+    print(f'\nRunning proposed model for {directory_name}/{file_name}, MD5 hash:{new_md5}')
+    new_predictions = run_xija_model(new_spec, tstart, tstop, msid, init)
+
+    sum_of_differences = sum(current_predictions.mvals - new_predictions.mvals)
```
Comment (taldcroft) [inline chandra_models/compare_models.py:116]: This should be `np.allclose(...)`. The `sum` can accidentally end up as zero.

**Review by taldcroft — COMMENTED** [review]
This new script is fine (and I wouldn't bother making the get_xija_model_spec change here, but be aware of it going forward), but I'd rather have the script be added via a separate PR from the change to the model spec file.

**Pull request merged**